src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPLIT_CHARACTER = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        File toFile = new File(toFileName);
+        StringBuilder builder = new StringBuilder();
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFile))) {
```

Comment:
where `FROM_FILE_NAME` is a String class-level constant
```suggestion
        try (BufferedReader reader = new BufferedReader(new FileReader(FROM_FILE_NAME))) {
```